lib/svtplay_dl/utils/retriever_multi.py

```diff
@@ -0,0 +1,111 @@
+# ex:ts=4:sw=4:sts=4:et
+# -*- tab-width: 4; c-basic-offset: 4; indent-tabs-mode: nil -*-
+
+import pycurl
+from io import BytesIO
+import time
+from datetime import timedelta
+
+# We should ignore SIGPIPE when using pycurl.NOSIGNAL - see
+# the libcurl tutorial for more info.
+try:
+    import signal
+    from signal import SIGPIPE, SIG_IGN
+except ImportError:
+    pass
+else:
+    signal.signal(SIGPIPE, SIG_IGN)
+
+from svtplay_dl.utils.output import progressbar, progress_stream, ETA, output
+
+FIREFOX_UA = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/45.0.2454.101 Safari/537.3'
+DEBUG = False
+
+
+def get(urls, cookies, silent=False, live=False, total_duration=None, tot_urls=0, num_conn=100):
+
+    # Check args
+    num_urls = len(urls)
+    num_conn = min(num_conn, num_urls)
+    eta = ETA(num_urls)
+
+    if DEBUG:
+        print("PycURL %s (compiled against 0x%x)" % (pycurl.version, pycurl.COMPILE_LIBCURL_VERSION_NUM))
+        print("----- Getting", num_urls, "URLs using", num_conn, "connections -----")
+
+    start_time = time.time()
+
+    # Pre-allocate a list of curl objects
+    m = pycurl.CurlMulti()
+    m.handles = []
+    for i in range(num_conn):
+        c = pycurl.Curl()
+        c.b = None
+        c.setopt(pycurl.FOLLOWLOCATION, 1)
+        c.setopt(pycurl.MAXREDIRS, 5)
+        c.setopt(pycurl.CONNECTTIMEOUT, 30)
+        c.setopt(pycurl.TIMEOUT, 300)
+        c.setopt(pycurl.NOSIGNAL, 1)
+        c.setopt(pycurl.HTTPHEADER, ["User-Agent:" + FIREFOX_UA])
```

Comment:
we should import the headers from`config.get("http_headers")`

Reply:
Done